business-central-parent/pom.xml

```diff
@@ -51,10 +51,10 @@
     <pluginManagement>
       <plugins>
         <plugin>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>findbugs-maven-plugin</artifactId>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
           <configuration>
-            <excludeFilterFile>${project.basedir}/../../src/main/findbugs/findbugs-exclude.xml</excludeFilterFile>
+            <excludeFilterFile>${project.basedir}/../../src/main/spotbugs/spotbugs-exclude.xml</excludeFilterFile>
```

Comment:
do we need this exclusion in distribution?
I removed that filtering and it works without any filter

Reply:
@mbiarnes  ^^^ have you tried to remove completely the spotbugs plugin here? Do we need it in distribution?